int8 drone_id
crazyflie_trajectory_manager/ConfigTrajectoryVectorPayload payload
---
bool proxy_success
bool result